nimble-newts/README.md

```diff
@@ -1,2 +1,18 @@
 # Nimble Newts
-This is the default README of your team's project. Please replace this by a README with more information on your project. At the very least, your README should contain information on how to set-up and run your project.
+
+A simple bot that responds to requests for Wikipedia articles.
+
+## How to run:
```

Comment:
This how to run is pretty vague, it could do with a bit more of a description about what the project is (and again, screenshots!)